GameBoard/Models/GameEvent/CreateGameEventViewModel.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using GameBoard.LogicLayer.GameEvents.Dtos;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GameBoard.Models.GameEvent
+{
+    public class CreateGameEventViewModel
```

Comment:
That's not true, the `Edit` one also has a default ctor. That's because in ASP.NET Core "Model bound complex types must not be abstract or value types and must have a parameterless constructor." Both of these are returned from the View via form, so they have to have a parameterless ctor and getters and setters on all properties used in the form.

Reply:
Thanks.